srml/support/src/traits.rs

```diff
@@ -181,7 +190,12 @@ pub trait LockableCurrency<AccountId>: Currency<AccountId> {
 	/// The quantity used to denote time; usually just a `BlockNumber`.
 	type Moment;
 
-	/// Introduce a new lock or change an existing one.
+	/// Create a new `BalanceLock` struct on account `who`.
```

Comment:
> Create a new `BalanceLock` struct on account `who`.

`BalanceLock` struct is balances module implementation details. Other implementations can be different.

Should be just:
> Create a new balance lock on account `who`